package.json

```diff
@@ -2,24 +2,33 @@
   "name": "pegasys-orchestrate-quick-start",
   "version": "2.0.0",
   "description": "PegaSys Orchestrate Quick start",
+  "config": {
+    "endpoints": {
+      "chainregistry": "localhost:8081",
+      "contractregistry": "localhost:8020",
+      "kafka": "localhost:9092"
+    }
+  },
   "scripts": {
     "lint": "tslint -p . -c tslint.json",
     "lint:fix": "npm run lint -- --fix",
     "format": "prettier --write 'src/**/*.{ts,js,json}'",
     "check-formatting": "prettier --list-different 'src/**/*.{ts,js,json}'",
     "clean": "rimraf ./dist ./build",
     "compile": "truffle compile",
-    "register-chain": "ts-node src/register-chain",
-    "register-contract": "orchestrate contracts register -e localhost:8020 -n Counter -f build/contracts/Counter.json",
-    "get-contract": "orchestrate contracts contract -e localhost:8020 -n Counter",
-    "get-catalog": "orchestrate contracts catalog --endpoint localhost:8020",
-    "generate-account": "orchestrate accounts generate --endpoint localhost:9092 --chain rinkeby",
+    "register-chain": "dotenv -- cross-var curl -X POST --data '{\"name\": \"%CHAIN%\", \"urls\":[\"%NETWORK_ENDPOINT%\"]}' ${npm_package_config_endpoints_chainregistry}/chains",
+    "get-latest-block": "dotenv -- cross-var curl -X POST --data '{\"jsonrpc\":\"2.0\",\"method\":\"eth_getBlockByNumber\",\"params\":[\"latest\", false],\"id\":1}' ${npm_package_config_endpoints_chainregistry}/%CHAIN_UUID%",
```

Comment:
It's used on line 216 of the readme to check that the chain is correctly proxied by Orchestrate. It's just an RPC request, the same that was in the readme but here it's encapsulated in an npm command for consistency and clarity.

Reply:
Sorry, you are right, it was collapse by default and I skipped. I think then it is okay for now, although I would rather push for having actual JS examples in the future instead of `curl` requests